### DISCUSSION TITLE
How to Retrieve Follower Count, Like Count, and Video Like Counts ?

### DISCUSSION THREAD

**Discussion by bunyaminbilenkaratas** [davidteather/TikTok-Api#1161] [Q&A] [answered]
Hello everyone,
Is there a way to get the follower count, like count for a user? and the like counts of videos ? I understand that this library does not support logged-in operations, but this data is displayed without logging in as well. Thank you in advance.

Best regards

**Comment by bunyaminbilenkaratas** [comment]
Still i have this problem

**Comment by joaovvbp** [comment, marked as answer]
You can, using `video.as_dict`, author information is in the `authorStats` and `author` key, you can look through the dictionary and find all the information available on that particular video, you can also provide the dictionary to some LLM like chatGPT and ask for help in retrieving whatever else you might need, it works pretty well for simple scripts.

This is an example of how I'm using it:

```
        author_data = {
            'author_unique_id': video_data_raw['author']['uniqueId'],
            'author_url': f"https://www.tiktok.com/@{video_data_raw['author']['id']}",
            'digg_count': video_data_raw['authorStats']['diggCount'],
            'follower_count': video_data_raw['authorStats']['followerCount'],
            'following_count': video_data_raw['authorStats']['followingCount'],
            'friend_count': video_data_raw['authorStats']['friendCount'],
            'heart_count': video_data_raw['authorStats']['heartCount'],
            'video_count': video_data_raw['authorStats']['videoCount']
        }
```

Have fun!

**Reply by bunyaminbilenkaratas** [reply]
Thank you.
